src/registrar/forms/application_wizard.py

```diff
@@ -585,12 +585,30 @@ class OtherContactsForm(RegistrarForm):
         error_messages={"required": "Enter a phone number for this contact."},
     )
 
+    # Override clean in order to correct validation logic
+    def clean(self):
+        # NOTE: using self.cleaned_data directly apparently causes a CORS error
+        cleaned = super().clean()
+        form_is_empty = all(v is None or v == "" for v in cleaned.values())
+        if form_is_empty:
+            # clear any errors raised by the form fields
+            # (before this clean() method is run, each field
+            # performs its own clean, which could result in
+            # errors that we wish to ignore at this point)
+            #
+            # NOTE: we cannot just clear() the errors list.
+            # That causes problems.
+            for field in self.fields:
+                if field in self.errors:
+                    del self.errors[field]
+        return cleaned
+
```

Comment:
(General comment/No action needed) I wonder if there is a place we can intercept this clean() call, before it is called. I.e. something like 
```
if not form_is_empty:
    self.clean()
```

Reply:
I think checking the values after clean is much better because they need to be resolved from a more complex set of data from Post.  Seemed easier and cleaner this way than trying to do something heavier further up the chain of events.